doc/u-boot-env.txt

```diff
@@ -1,54 +1,46 @@
 Boot storage device setting
 ===================================
-In order to make it possible to boot from various storage devices,
```

Comment:
this section becomes empty - should it be removed?

Reply:
Thanks, good comment. I have added information on possible settings for 'bootcmd' and add reason for using 'z_bootcmd_help_*' to this section.